.prettierrc

```diff
@@ -0,0 +1,5 @@
+printWidth: 100
```

Comment:
this can not be extended from ocular?

Reply:
Maybe, we will need to switch to use JSON format though.